tests/test_vbox.py

```diff
@@ -0,0 +1,22 @@
+from io import StringIO
+
+from dissect.hypervisor.disk.vdi import Vbox
+
+
+def test_vbox():
+
+    xml = """<?xml version="1.0"?>
+                <VirtualBox xmlns="http://www.virtualbox.org/">
+                    <Machine>
+                        <MediaRegistry>
+                            <HardDisks>
+                                <HardDisk location="os2warp4.vdi" format="VDI" type="Normal" />
+                            </HardDisks>
+                        </MediaRegistry>
+                    </Machine>
+                </VirtualBox>
+    """
+
+    with StringIO(xml) as fh:
```

Comment:
```suggestion
    xml = """
    <?xml version="1.0"?>
    <VirtualBox xmlns="http://www.virtualbox.org/">
        <Machine>
            <MediaRegistry>
                <HardDisks>
                    <HardDisk location="os2warp4.vdi" format="VDI" type="Normal" />
                </HardDisks>
            </MediaRegistry>
        </Machine>
    </VirtualBox>
    """

    with StringIO(xml.strip()) as fh:
```
Maybe personal preference but looks a bit nicer :)

Reply:
That will make the xml invalid and the test will fail.

Reply:
That will make the xml invalid and the test will fail.

Reply:
The `.strip()` will take care of that right?